README.rst

```diff
@@ -37,55 +37,44 @@ Usage Example
 .. code:: python
 
     import time
+    import random
     import board
     import terminalio
     from adafruit_matrixportal.matrixportal import MatrixPortal
 
-    # You can display in 'GBP', 'EUR' or 'USD'
-    CURRENCY = "USD"
-    # Set up where we'll be fetching data from
-    DATA_SOURCE = "https://api.coindesk.com/v1/bpi/currentprice.json"
-    DATA_LOCATION = ["bpi", CURRENCY, "rate_float"]
+    # --- Display setup ---
+    matrixportal = MatrixPortal(status_neopixel=board.NEOPIXEL, debug=True)
 
-
-    def text_transform(val):
-        if CURRENCY == "USD":
-            return "$%d" % val
-        if CURRENCY == "EUR":
-            return "‎€%d" % val
-        if CURRENCY == "GBP":
-            return "£%d" % val
-        return "%d" % val
-
-
-    # the current working directory (where this file is)
-    cwd = ("/" + __file__).rsplit("/", 1)[0]
-
-    matrixportal = MatrixPortal(
-        url=DATA_SOURCE,
-        json_path=DATA_LOCATION,
-        status_neopixel=board.NEOPIXEL,
-        debug=True,
+    # Create a new label with the color and text selected
+    matrixportal.add_text(
+        text_font=terminalio.FONT,
+        text_position=(0, (matrixportal.graphics.display.height // 2) - 1),
+        scrolling=True,
     )
 
+    # Static 'Connecting' Text
     matrixportal.add_text(
         text_font=terminalio.FONT,
-        text_position=(16, 16),
-        text_color=0xFFFFFF,
-        text_transform=text_transform,
+        text_position=(2, (matrixportal.graphics.display.height // 2) - 1),
     )
-    matrixportal.preload_font(b"$012345789")  # preload numbers
-    matrixportal.preload_font((0x00A3, 0x20AC))  # preload gbp/euro symbol
 
+    SCROLL_DELAY = 0.03
+
+    contents = [
+        { 'text': 'THIS IS RED',  'color': '#cf2727'},
+        { 'text': 'THIS IS BLUE', 'color': '#0846e4'},
+    ]
+
+    matrixportal.set_text(" ", 1)
```

Comment:
We might be able to remove this line? I'm not sure if it's needed. The script seems to behave the same without it.

Reply:
Will do.  Copy/paste from previous examples.